tso-server/src/main/java/org/apache/omid/tso/DisruptorModule.java

```diff
@@ -18,14 +18,38 @@
 package org.apache.omid.tso;
 
 import com.google.inject.AbstractModule;
+import com.google.inject.name.Names;
+import com.lmax.disruptor.BlockingWaitStrategy;
+import com.lmax.disruptor.BusySpinWaitStrategy;
+import com.lmax.disruptor.WaitStrategy;
 
 import javax.inject.Singleton;
 
 public class DisruptorModule extends AbstractModule {
 
+    private final TSOServerConfig config;
+
+    public DisruptorModule(TSOServerConfig config) {
+        this.config = config;
+    }
+
     @Override
     protected void configure() {
-
+        switch (config.getWaitStrategyEnum()) {
+        // A low-cpu usage Disruptor configuration for using in local/test environments
+        case LOW_CPU:
+             bind(WaitStrategy.class).annotatedWith(Names.named("PersistenceStrategy")).to(BlockingWaitStrategy.class);
+             bind(WaitStrategy.class).annotatedWith(Names.named("ReplyStrategy")).to(BlockingWaitStrategy.class);
+             bind(WaitStrategy.class).annotatedWith(Names.named("RetryStrategy")).to(BlockingWaitStrategy.class);
```

Comment:
I've realized that this should be a `YieldingWaitStrategy` according to the current code (see below). So this should be:

```
  bind(WaitStrategy.class).annotatedWith(Names.named("RetryStrategy")).to(YieldingWaitStrategy.class);
```